05/app.js

```diff
@@ -9,6 +9,36 @@ const stats = {
 
 /* tutaj umieść swój kod */
 
+const pElements = document.querySelectorAll('p')
+
+const countClick = function(e) {
+    e.preventDefault();
+    const elTagName = e.target.tagName;
+    
+    if (elTagName === 'A') {
+        // console.log(`kliknales w link`)
+        let aEl = e.target.getAttribute('href')
+        if(typeof stats.links[aEl] === 'undefined') {
+            stats.links[aEl] = 1;
+        } else {
+            stats.links[aEl] += 1;
+        }  
```

Comment:
Zwróć uwagę, że w linii 29-34 dzieją się podobne rzeczy.
Postaraj się napisać funkcję dla tych obu miejsc -  poszukaj elementy zmieniające się i to będą parametry funkcji, sama funkcja ma dokonywać zmian w statystykach w zależności od parametrów :)